docs-ref-conceptual/azure-cli-endpoints.md

```diff
@@ -0,0 +1,220 @@
+postgresql_server_endpoint | *.postgres.database.chinacloudapi.cn
+mariadb_server_endpoint | *.mariadb.database.chinacloudapi.cn
+acr_login_server_endpoint | *.azurecr.cn
+synapse_analytics_endpoint | *.dev.azuresynapse.azure.cn
+
+### [Azure Germany Cloud](#tab/azure-germany-cloud)
+
+### Endpoints
+
+|Endpoint group | Endpoint
+|-|-|
+management | https://management.core.cloudapi.de/
+resource_manager | https://management.microsoftazure.de
+sql_management | https://management.core.cloudapi.de:8443/
+batch_resource_id | https://batch.cloudapi.de/
+gallery | https://gallery.cloudapi.de/
+active_directory | https://login.microsoftonline.de
+active_directory_resource_id | https://management.core.cloudapi.de/
+active_directory_graph_resource_id | https://graph.cloudapi.de/
+microsoft_graph_resource_id | https://graph.microsoft.de
+vm_image_alias_doc | https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/arm-compute/quickstart-templates/aliases.json
+media_resource_id | https://rest.media.cloudapi.de
+ossrdbms_resource_id | https://ossrdbms-aad.database.cloudapi.de
+portal | https://portal.microsoftazure.de
+
+### Endpoint suffixes
+
+|Suffix name | Suffix
+|-|-|
+storage_endpoint | *.core.cloudapi.de
+keyvault_dns | *.vault.microsoftazure.de
+mhsm_dns | *.managedhsm.microsoftazure.de
+sql_server_hostname | *.database.cloudapi.de
+mysql_server_endpoint | *.mysql.database.cloudapi.de
+postgresql_server_endpoint | *.postgres.database.cloudapi.de
+mariadb_server_endpoint | *.mariadb.database.cloudapi.de
+
+---
+
+## Extension endpoints
+
+Azure CLI extensions are optional and installed separately.  All extensions install with endpoint **azcliprod.blob.core.windows.net** with the following exceptions:
+
+|Extension name | Endpoint
+|-|-|
+| arcappliance | arcplatformcliextprod.blob.core.windows.net
+| customlocation |
+| arcdata | azurearcdatacli.blob.core.windows.net
+| azure-batch-cli-extensions | github.com\Azure
+| azure-devops |
+| azure-iot |
+| baremetal-infrastructure |
+| csvmware |
+| deploy-to-azure |
+| sap-hana |
+| azure-cli-ml | azurecliext.blob.core.windows.net
+| azurestackhci | hybridaksstorage.z13.web.core.windows.net
+| Hybridaks |
+| fzf | pahealyfzf.blob.core.windows.net
+|image-copy-extension | files.pythonhosted.org
+| managementpartner |
+| resource-graph |
+| ml | azuremlsdktestpypi.blob.core.windows.net
+| qbs | qbsazcliextension.blob.core.windows.net
+
+To see a list of installed extension for the Azure CLI, enter `az` at your terminal command prompt and press **enter**.  To see a list of available extensions see [Available Azure CLI extensions](/cli/azure/azure-cli-extensions-list).
+
+## Engineering questions
+1. What about **pypi.org** and **files.pythonhosted.org**?  According to the customer request for installing the `connectedk8s` extension, they had to allow these two URLs.  I only see **files.pythonhosted.org** under `image-copy-extension`. **pypi.org** isn't in any list.
```

Comment:
```suggestion

1. What about **pypi.org** and **files.pythonhosted.org**?  According to the customer request for installing the `connectedk8s` extension, they had to allow these two URLs.  I only see **files.pythonhosted.org** under `image-copy-extension`. **pypi.org** isn't in any list.
```

Reply:
@chasewilson , @Kairu-ms, This H2 isn't part of the article.  It is here as a question to Azure CLI engineering.  Where do **pypi.org** and **files.pythonhosted.org** belong in this article?

Reply:
**pypi.org** and **files.pythonhosted.org** will be used to download dependences.  It is the standard repository for Python packages and tools.